app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -16,8 +16,8 @@
 
   hide_heading ||= false
 
-  email_signup_link_text_locale = local_assigns[:email_signup_link_text_locale]
-  feed_link_text_locale = local_assigns[:feed_link_text_locale]
+  email_signup_link_text_locale = local_assigns[:email_signup_link_text_locale].present? ? local_assigns[:email_signup_link_text_locale] : nil
```

Comment:
You could shorten this with [`.presence`](https://apidock.com/rails/Object/presence) to `local_assigns[:email_signup_link_text_locale].presence` (although it might not be clearer)

Reply:
Done - thanks for the suggestion.